ocfweb/docs/docs/staff/shorturls.md

```diff
@@ -0,0 +1,8 @@
+[[!meta title="Shorturls"]]
+
+We use `ocf.io` as a URL shortener, in addition to being an alias for
+`ocf.berkeley.edu`.
+
+You can find a list of `ocf.io` shortlinks here:
+
+https://github.com/ocf/puppet/blob/master/modules/ocf_www/manifests/site/shorturl.pp
```

Comment:
I'd put this as a link from some text in the sentence above, instead of just a plain URL like this, just to make it more readable for the web side of things.